examples/communication/relay/ReceiveUserDataRelaySample/ReadMe.txt

```diff
@@ -0,0 +1,82 @@
+  Introduction
+  ------------
+  This sample C# application shows how to receive and process data coming
+  from the bluetooth interface of the XBee device using a callback executed
```

Comment:
I would say "from other interfaces of the XBee device", the example is generic for any interface.